myuw/static/css/mobile.less

```diff
@@ -553,20 +553,21 @@ label { font-weight: normal; }
 			}
 		}
 	}
-	.message-dialog {
-		.modal-content {
-			h3 {margin-bottom: 10px;font-family: 'encode_sans_normalbold', sans-serif; font-weight: 600; font-size: 18px;color: #85754d;}
-			.modal-header {border-bottom: none;padding-bottom: 0;}
-			.modal-body {padding: 0 15px;
-				ul {
-					li {line-height: 25px;}
-				}
+}
+.message-dialog {
+	.modal-content {
+		h3 {margin-bottom: 10px;font-family: 'encode_sans_normalbold', sans-serif; font-weight: 600; font-size: 18px;color: #85754d;}
+		.modal-header {border-bottom: none;padding-bottom: 0;}
+		.modal-body {padding: 0 15px;
```

Comment:
Not a blocker, but if we're editing this code anyway we could update the padding/margins to fall into our 8px/16px system.